.github/workflows/ci.yml

```diff
@@ -204,3 +204,11 @@ jobs:
       - name: errors
         run: golangci-lint run
         if: ${{ failure() }}
+  check-required-label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: mheap/github-action-required-labels@v5
+        with:
+          mode: exactly
+          count: 1
+          labels: "ignore-for-release, feature, enhancement, maintenance, docs, i18n"
```

Comment:
Isn't "bug" missing here?

Reply:
Ah, yes it is haha

Reply:
https://github.com/jesseduffield/lazygit/pull/2798